cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -120,6 +123,54 @@ function ManagementCredentials(props: Partial<StepProps>) {
             }
         }
     };
+
+    const isConnected = () => {
+        if (connectionStatus === CONNECTION_STATUS.CONNECTED) {
```

Comment:
nit: I think this could be expressed as: `return connectionStatus === CONNECTION_STATUS.CONNECTED`

Reply:
Yes sir. That is more clean. Correct that